cmd/channel/email/main.go

```diff
@@ -5,31 +5,39 @@ import (
 	"database/sql"
 	"encoding/json"
 	"fmt"
+	"github.com/emersion/go-sasl"
+	"github.com/emersion/go-smtp"
 	"github.com/icinga/icinga-notifications/internal"
 	"github.com/icinga/icinga-notifications/pkg/plugin"
 	"github.com/icinga/icingadb/pkg/types"
+	"github.com/jhillyerd/enmime"
 	"net"
-	"net/smtp"
+	"net/mail"
 	"os"
 	"os/user"
-	"strings"
+	"time"
 )
 
 type Email struct {
-	Host string `json:"host"`
-	Port string `json:"port"`
-	From string `json:"from"`
+	Host       string `json:"host"`
+	Port       string `json:"port"`
+	SenderName string `json:"sender_name"`
+	SenderMail string `json:"sender_mail"`
+	From       string `json:"from"`
```

Comment:
`From` looks unused. Was it replaced completely by these other two options?

Reply:
Yes, I forgot to remove it from here.